src/M6Web/Component/RedisMock/RedisMock.php

```diff
@@ -79,6 +79,22 @@ public function get($key)
 
     public function set($key, $value, $seconds = null)
     {
+        if (\is_array($seconds)) {
+            $options = $seconds;
+            if (\in_array('nx', $options, true) && $this->get($key)) {
+                return $this->returnPipedInfo(0);
+            }
+            if (\in_array('xx', $options, true) && !$this->get($key)) {
+                return $this->returnPipedInfo(0);
+            }
+
+            $seconds = null;
+            if (isset($options['ex'])) {
+                $seconds = $options['ex'];
+            } elseif (isset($options['px'])) {
+                $seconds = $options['px'] / 1000;
```

Comment:
Can you add a unit to ease maintenance? Something like `1000 /* ms per seconds*/`?
(In don't know if my example is correct, it's just an example ^^)

Reply:
Well I don't know why the build fails now when all I did was add a comment block...